map-widget/qmlmapwidgethelper.cpp

```diff
@@ -261,7 +261,9 @@ QString MapWidgetHelper::pluginObject()
 			      "    Component.onCompleted: {"
 			      "        if (availableServiceProviders.indexOf(name) === -1) {"
 			      "            console.warn('MapWidget.qml: cannot find a plugin named: ' + name);"
+			      "            console.warn('Test van Egbert');"
```

Comment:
I'm not sure we need that in the final code :)

Reply:
I know, I already made a commit that removed that ;-) I overlooked it this morning